mailbot/mailbot.py

```diff
@@ -59,7 +59,14 @@ def process_messages(self):
 
         for uid, msg in messages.items():
             self.mark_processing(uid)
-            message = message_from_string(msg['RFC822'])
+
+            # Patch required for Python 3 compatibility:
+            # In Python 3.x, IMAPClient returns byte arrays instead of str
+            if 'RFC822' in msg:
```

Comment:
Why not use something like that: https://github.com/Kinto/kinto/blob/master/kinto/core/utils.py#L343-L353